onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -77,11 +77,22 @@ class TorchLibOpInfo:
     nondeterministic: bool = False
     # Whether the function is designed for complex inputs
     complex: bool = False
+    # The tolerance for the test {dtype: (rtol, atol)}.
```

Comment:
nit, change to:

The tolerance of the inference result difference between PyTorch and ORT. Format: {dtype: (rtol, atol)}.
For example: {torch.float16: (1e-3, 1e-3)}

Reply:
Done